src/stories/Header/component.stories.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+import StyledHeader from "./index";
+
+export default {
+    title: "Header",
+    component: StyledHeader,
+};
+
+export const Default = () => <StyledHeader></StyledHeader>;
```

Comment:
No need to export. You should have 
storiesOf('StyledHeader component', StyledHeader)
.add('Scenario 1', () => <StyledHeader {...props}>)
.add('Scenario 2', () => <StyledHeader ...
.add('Scenario 1', () => <StyledHeader ...

Reply:
done